html_css/typography.md

```diff
@@ -20,8 +20,9 @@ Do what you can to get at least a familiarity with the high level decisions invo
 
 <div class="lesson-content__panel" markdown="1">
 1. Read [Shay Howe on Fonts and Typography](http://learn.shayhowe.com/html-css/typography)
-2. Browse through [A Beginner's Guide to Using Google Web Fonts by DesignShack](http://designshack.net/articles/css/a-beginners-guide-to-using-google-web-fonts/), but don't get too lost in the details.
-3. Browse the [Google Web Fonts](http://www.google.com/fonts) and see if any of them seem interesting to you.
+2. Read this article about Serif and Sans Serif fonts [Sans Serif vs Serif Font: Which Should You Use & When?](https://www.impactplus.com/blog/sans-serif-vs-serif-font-which-should-you-use-when)
```

Comment:
I think that this would be best under the Additional Resources section. Could you please move it there.

Reply:
Disregard review. I will merge this as it is in the learning outcomes